epub33/reports/a11y-properties-use.md

```diff
@@ -0,0 +1,104 @@
+# EPUB Accessibility 1.1 Metadata Usage Report
+
+## Candidate Recommendation Exit Criteria
+
+The EPUB Working Group intends to exit the Candidate Recommendation stage and submit
+the EPUB Accessibility 1.1 specification for consideration as a W3C Proposed Recommendation
+after documenting implementation of each feature.</p>
+
+For this specification to advance to Proposed Recommendation, it has to be
+proven that metadata defined and required in this specification have sufficient usage by the
+target communities. This metadata falls into two categories:</p>
+
+1. required and recommended schema.org metadata for describing the accessibility
+of publications; and
+2. EPUB-defined metadata properties for reporting aspects of conformance
+
+Usage of these properties means that they are regularly included in the Package Document
+metadata for their EPUB Publications (as appropriate for each title).
+
+## Publisher Implementations
+
+### Schema.org discovery metadata
+
+The following table provides a list of publishers who have stated that they are currently using
+the schema.org discovery metadata properties in production or who are in the process of rolling
+out their implementations.
+
+<table>
+    <thead>
+        <tr>
+            <th>Role</th>
+            <th>[Publisher]</th>
```

Comment:
Why square brackets? I don't see what they're trying to indicate.

Reply:
It's a placeholder right now for an actual publisher name. Each column will list their usage. I can add an entry for Benetech's GCA program so we don't need the placeholder if that's okay with you @clapierre ?